src/sections/Join/index.js

```diff
@@ -1,9 +1,44 @@
-import React from 'react';
+import React, { useState } from 'react';
 
+import { FormButton } from 'Elements';
+import {
+  joinLayout, formBlock,
+  formTitle, formSubtitle, formButton,
+  joinForm, inputFields, cta, ctaText, ctaLogo
+} from './style.css';
+
+import GitterLogo from 'Elements/GitterLogo';
 
 const Join = () => {
-  return(
-    <div>JOIN</div>
+  const [email, setEmail] = useState('');
+  
+  function handleSubmit(event) {
+    alert('form submitted');
+    event.preventDefault();
+  }
+
+  return (
+    <section id="join" className={joinLayout}>
+      <div className={formBlock}>
+        <h1 className={formTitle}>Join</h1>
+        <span className={formSubtitle}>Subscribe to monthly meetup events & get notified of upcoming events like community workshops, hackathons, presentations and recordings.</span>
+
+        <form id="joinForm" className={joinForm} onSubmit={handleSubmit}>
+          <fieldset className={inputFields}>
+            <input id='joinEmail' type="text" value={email} onChange={ev => setEmail(ev.target.value)} required />
+          </fieldset>
+          <FormButton className={formButton}>Subscribe</FormButton>
```

Comment:
pull master and let me know if the `FormSubmitButton` works

Reply:
Changed this up, and yes the new button component worked and required less changes.

I think this PR is ready.  Let me know if you need any more help with your meetup or anything else.